lib/apkstats/plugin.rb

```diff
@@ -220,6 +244,22 @@ def target_sdk(_opts = {})
       run_command(__method__)
     end
 
+    # Show the methods reference count of your apk file.
+    #
+    # @return [Fixnum] return positive value if exists, otherwise -1.
+    def reference_count(_opts = {})
```

Comment:
Could you please use `method_reference_count` instead? because there are other *references*  like fields, constructors etc.

Reply:
Okay, I'll fix it.